packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -186,14 +194,7 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
   }
 
   private _setDistanceFromTop(distance: number): void {
```

Comment:
>_setDistanceFromTop [](start = 10, length = 19)

We are mostly setting 'distanceFromTop' directly in state at other places as well now. We can remove this. There is only once consumer for this anyways.